### PR TITLE
FMO-63: FMO tool documentation

### PR DESCRIPTION
**Pull request by jsvapiav**
Add fmo-tool documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,370 @@
-# fmo-tool
\ No newline at end of file
+# fmo-tool
+FMO-os system managment tool
```
Comment (buianhhuy96) [inline README.md]: typo: management

File: README.md [hunk 2]
```diff
@@ -1 +1,370 @@
...
+if you want to apply / play with those rules manually:
+```bash
+$ fmo-tool dpf rules netvm -v
+...
+iptables -I INPUT -p udp --dport 123 -j ACCEPT
+iptables -t nat -I PREROUTING -p udp -d 192.168.101.11 --dport 123 -j DNAT --to-detination NA:123
+iptables -I INPUT -p tcp --dport 123 -j ACCEPT
+iptables -t nat -I PREROUTING -p tcp -d 192.168.101.11 --dport 123 -j DNAT --to-detination NA:123
+iptables -I INPUT -p tcp --dport 42 -j ACCEPT
+iptables -t nat -I PREROUTING -p tcp -d 192.168.101.2 --dport 42 -j DNAT --to-detination NA:22
```
Comment (buianhhuy96) [inline README.md]: typo: destination

File: README.md (hunk 1 above)
Comment (jsvapiav) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (jsvapiav) [reply README.md]: Fixed

**Review by buianhhuy96 — APPROVED** [review]
LGTM

**Pull request merged**